pymc/sampling/mcmc.py

```diff
@@ -212,9 +212,79 @@ def all_continuous(vars):
         return True
 
 
+def sample_external_nuts(
+    sampler="nutpie",
+    draws: int = 1000,
+    initvals: Optional[Union[StartDict, Sequence[Optional[StartDict]]]] = None,
+    chains: Optional[int] = None,
+    cores: Optional[int] = None,
+    tune: int = 1000,
+    progressbar: bool = True,
+    model=None,
+    random_seed: RandomState = None,
+    discard_tuned_samples: bool = True,
+    **kwargs,
+):
+    target_accept = kwargs.get("nuts", {}).get("target_accept", 0.8)
+    # Try different fast samplers in order of their speed
+    if sampler == "nutpie":
+        try:
+            import nutpie
+        except ImportError:
+            _log.info("nutpie not found. Install it with mamba install -c conda-forge nutpie")
+
+        if (draws == 1000) and (tune == 1000):
```

Comment:
Is this important? If so you should use None as the default argument for these variables, otherwise you will be ignoring when users explicitly request 1000 draws and tune.

Reply:
But this is just forwarded for sample() so we'd need to change it there.